app/src/main/res/layout/activity_photo_view.xml

```diff
@@ -20,7 +20,7 @@
         android:visibility="visible" />
 
 
-    <ImageView
+    <uk.co.senab.photoview.PhotoView
```

Comment:
Works well, but what's with this name` uk.co.senab.photoview.PhotoView` shouldn't it be com.github.chrisbanes:PhotoView ?